components/dashboard/src/data/git-providers/search-repositories-query.ts

```diff
@@ -24,7 +24,7 @@ export const useSearchRepositories = ({ searchString, limit }: { searchString: s
             return repositories;
         },
         {
-            enabled: !!org && debouncedSearchString.length >= 3,
+            enabled: !!org && debouncedSearchString.length > 0,
```

Comment:
@filiptronicek Did you experiment with the min length being 2 instead of 1? :thinking: 
Just trying to understand if we might hit other problems here... e.g., rate limits etc. :thinking:

Reply:
I didn't, mostly because 2 felt even more arbitrary than 1 😄. Hopefully, since we debounce the search string, rate limits shouldn't be too big of an issue(?)

Reply:
Fine with me.

Was thinking about clients with huge list of repositories... but also, I think we fixed those issues separately. So fine with me. If we see issues, we can quickly adjust. :+1: